### ISSUE TITLE
pg_schema doesn't work with pg_operator

### ISSUE THREAD

**Issue by neilyio** [pgcentralfoundation/pgrx#1301]
It doesn't seem like pgrx is creating `pg_operator` functions according to`pg_schema`. I'm able to reproduce this with a minimal `pgrx` setup, where I try to define an operator in a `pg_catalog` module to make it available in the global Postgres namespace.

Setup with:
```
cargo pgrx new pgrx-repro
```

Set schema to `repro` in `rgrx_repro.control`:
```
comment = 'pgrx_repro:  Created by pgrx'
default_version = '@CARGO_VERSION@'
module_pathname = '$libdir/pgrx_repro'
relocatable = false
superuser = true
schema = repro
```

In `src/lib.rs`, add the following:
```rust

#[pg_schema]
mod pg_catalog {
    use pgrx::{opname, pg_operator};

    #[pg_operator]
    #[opname(==>)]
    fn concat_strings(left: String, right: String) -> String {
        left + &right
    }
}
```

Test with `cargo pgrx run --pgcli`:
```sql
pgrx_repro> create extension pgrx_repro;
CREATE EXTENSION

pgrx_repro> select 'hello' ==> 'world'
operator does not exist: unknown ==> unknown
LINE 1: select 'hello' ==> 'world'
                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.

pgrx_repro> SELECT set_config('search_path', 'repro,'||current_setting('search_path'), false);
+-----------------------+
| set_config            |
|-----------------------|
| repro,"$user", public |
+-----------------------+
SELECT 1

pgrx_repro> select 'hello' ==> 'world'
+------------+
| ?column?   |
|------------|
| helloworld |
+------------+
SELECT 1
```

I expected the `==>` operator to be available right after `create extension pgrx_repro;`

**Comment by eeeebbbbrrrr** [comment]
Hey thanks.  Could you include the output of `cargo pgrx schema` for this too?

**Comment by neilyio** [comment]
Sure thing. `cargo pgrx schema` below:

```sql
/* 
This file is auto generated by pgrx.

The ordering of items is not stable, it is driven by a dependency graph.
*/

-- src/lib.rs:6
-- pgrx_repro::hello_pgrx_repro
CREATE  FUNCTION repro."hello_pgrx_repro"() RETURNS TEXT /* &str */
STRICT
LANGUAGE c /* Rust */
AS 'MODULE_PATHNAME', 'hello_pgrx_repro_wrapper';

-- src/lib.rs:16
-- pgrx_repro::pg_catalog::concat_strings
CREATE  FUNCTION pg_catalog."concat_strings"(
	"left" TEXT, /* alloc::string::String */
	"right" TEXT /* alloc::string::String */
) RETURNS TEXT /* alloc::string::String */
STRICT
LANGUAGE c /* Rust */
AS 'MODULE_PATHNAME', 'concat_strings_wrapper';

-- src/lib.rs:16
-- pgrx_repro::pg_catalog::concat_strings
CREATE OPERATOR ==> (
	PROCEDURE="concat_strings",
	LEFTARG=TEXT, /* alloc::string::String */
	RIGHTARG=TEXT /* alloc::string::String */
);
```

**Comment by eeeebbbbrrrr** [comment]
yup, sure enough.  we get the function in the right schema but not the operator.  dang.  

I'll look into this today.  Thanks for the report.

**Comment by eeeebbbbrrrr** [comment]
there's a fix for this up in https://github.com/pgcentralfoundation/pgrx/pull/1302.

We'll get it merged and committed soon.

**Comment by workingjubilee** [comment]
Released in v0.10.2!
